modules/elasticsearch/src/main/java/org/testcontainers/elasticsearch/TcpHost.java

```diff
@@ -0,0 +1,24 @@
+package org.testcontainers.elasticsearch;
+
+/**
+ * Created on 05/11/18.
+ *
+ * @author Reda.Housni-Alaoui
```

Comment:
Please could you remove this javadoc? We try to avoid these kind of generated javadoc comments.